.github/workflows/e2e-parallel-destroy.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         run: |
           terraform init -upgrade=true
           terraform destroy -target=module.eks_blueprints_kubernetes_addons  -no-color -input=false -auto-approve
-          terraform destroy -target=module.eks_blueprints  -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks_blueprints_addons -no-color -input=false -auto-approve
+          terraform destroy -target=module.eks_blueprints -no-color -input=false -auto-approve
```

Comment:
I know this should be already like this, but sure this will not break the workflow? Like something tied with the old name convention.

Reply:
This makes it more correct. Because of target destroy for the current patterns